cmd/aries-agentd/startcmd/start.go

```diff
@@ -9,6 +9,8 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
+	"os"
```

Comment:
@troyronda  @sudeshrshetty  @rolsonquadras I think we should use config file to pass the configuration instead of passing directly into CLI

Reply:
(already discussed in daily standup, won't be doing)